.github/workflows/bat.yml

```diff
@@ -26,34 +26,50 @@ jobs:
   integ:
     needs: bat
     runs-on: ${{ matrix.os }}
+    outputs:
+      matlab_root: ${{ steps.setup_matlab.outputs.MATLAB_ROOT }}
```

Comment:
I am thinking maybe we should change the output name to `matlabroot` anyways because this is the name of the MATLAB function that returns the same information.